components/k8s-model-server/testing/test-serving-container/components/workflows.libsonnet

```diff
@@ -0,0 +1,147 @@
+{
+  // TODO(https://github.com/ksonnet/ksonnet/issues/222): Taking namespace as an argument is a work around for the fact that ksonnet
+  // doesn't support automatically piping in the namespace from the environment to prototypes.
+
+  // convert a list of two items into a map representing an environment variable
+  listToMap:: function(v)
+    {
+      name: v[0],
+      value: v[1],
+    },
+
+  // Function to turn comma separated list of prow environment variables into a dictionary.
+  parseEnv:: function(v)
+    local pieces = std.split(v, ",");
+    if v != "" && std.length(pieces) > 0 then
+      std.map(
+        function(i) $.listToMap(std.split(i, "=")),
+        std.split(v, ",")
+      )
+    else [],
+
+  parts(namespace, name):: {
+    // Workflow to run the e2e test.
+    e2e(prow_env, bucket, serving_image):
+      // mountPath is the directory where the volume to store the test data
+      // should be mounted.
+      local mountPath = "/mnt/" + "test-data-volume";
+      // testDir is the root directory for all data for a particular test run.
+      local testDir = mountPath + "/" + name;
+      // outputDir is the directory to sync to GCS to contain the output for this job.
+      local outputDir = testDir + "/output";
+      local artifactsDir = outputDir + "/artifacts";
+      local srcDir = testDir + "/src";
+      // local testing_image = "gcr.io/mlkube-testing/kubeflow-testing";
+      local testing_image = "gcr.io/kai-test2/kubeflow-testing:1.0";
```

Comment:
Why is this point at kai-cloud?
If you need to override these values make the parameters and just override them per environment.

Reply:
I made some change to the Dockerfile (install docker).
Made it a param now.